### DISCUSSION TITLE
Error library(keras) in R

### DISCUSSION THREAD

**Discussion by zahrafromhome** [teamalgoritma/community#726] [Q&A] [answered]
Saya masih belum dapat menjalankan keras di R, berikut ini lampiran screenshot inclass keras final yang saya jalankan. Saya sudah mencoba untuk uninstall keras dan kemudian install kembali dengan `pip install tensorflow==2.12 --user` tapi masih error. Mohon bantuannya

![image](https://github.com/teamalgoritma/community/assets/82038584/486f1289-c8ea-480f-8541-42840ccea2fd)

**Comment by ramdhinafinita** [comment]
Dear Mba @zahrafromhome 
Berikut kami lampirkan cara untuk bisa dicoba:
https://github.com/rstudio/keras/issues/1136

**Reply by zahrafromhome** [reply]
instalasinya sudah berjalan dan tidak error kak, tapi saat run model sequential masih error 'Python module tensorflow.keras was not found.'

![image](https://github.com/teamalgoritma/community/assets/82038584/26b28980-728a-4640-89ba-74ff353de7b1)

**Reply by divakartika** [reply, marked as answer]
Dear @zahrafromhome 

Mohon maaf baru merespon kembali. Apakah sudah mengikuti langkah-langkah yang tertera pada **Algoritma Handbook Guide - Keras in R** secara berurutan? Dokumen tersebut terdapat di dalam folder material utama course Neural Network. Dapat dicoba kembali dengan membuat virtual environment baru dan mengikuti langkah-langkah yang tertera. 

Jika masih mengalami kendala yang sama, dapat dicoba menggunakan [Kaggle Notebook](https://www.kaggle.com/code) atau [Google Colaboratory](https://research.google.com/colaboratory/) sebagai alternatif. Tidak perlu melakukan instalasi library karena sudah disediakan oleh platform tersebut.

Semoga membantu :)

Regards,
Team Algoritma

**Reply by zahrafromhome** [reply]
saya sudah ikuti langkah-langkah dari handbook Keras in R, tapi masih error. Mungkin akan saya coba menggunakan Kaggle/Google Colab, terima kasih kak
